app/index.js

```diff
@@ -1 +1,9 @@
 import 'whatwg-fetch';
+import './data.json';
+import './twitch/add-games-to-list';
```

Comment:
When importing, make sure to `import variableName from './path-to-module';` or else you can't use the exported class, function, or JSON data from the imported file.